lib/ImageHelper.php

```diff
@@ -2,15 +2,8 @@
 
 namespace Timber;
 
-use Timber\Image;
-use Timber\Image\Operation\ToJpg;
```

Comment:
These `Timber\Image\Operation` declarations are redundant if you're also using the fully qualified class name as we are [here](https://github.com/timber/timber/blob/d3b301ff85507c2739a7ecbd686b69e0d5ca2f09/lib/ImageHelper.php#L684), but note you can also `use` just a namespace rather than a fully qualified class. So for example we can include a single `use Timber\Image\Operation` here and then usage is simply `new Operation\ToJpg()` or `new Operation\Letterbox`.

Reply:
Yes, of course! Makes total sense for `Timber\Image\Operation` 👍. I updated the class names in the `ImageHelper` class.